1-mandatory/1-writers.js

```diff
@@ -51,7 +51,9 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
+  for (author of writers) { 
```

Comment:
Looks good, just make sure you're using `let` or `const` when declaring the variable `author`